charts/prometheus-statsd-exporter/values.yaml

```diff
@@ -63,18 +63,37 @@ readinessProbe:
     path: /health
     port: http
 
+prometheus:
+  monitor:
+    enabled: false
+    additionalLabels: {}
+    namespace: ""
+    interval: 30s
+    # ApiVersion for the podMonitor Resource(defaults to "monitoring.googleapis.com/v1")
+    apiVersion: ""
+    ## resource kind(defaults to "PodMonitoring")
+    kind: ""
+    # metrics exposing endpoint
+    endpoint: "/metrics"
+
 serviceMonitor:
   enabled: false
   interval: 30s
   scrapeTimeout: 10s
   namespace: monitoring
   honorLabels: false
   additionalLabels: {}
+  ## resource kind(defaults to "PodMonitoring")
```

Comment:
this seems wrong

Reply:
fixed it, 
the description was wrong but[ serviceMonitor resource ](https://github.com/prometheus-community/helm-charts/pull/2596/files#diff-541ccd8e525b85e105d07f955e7b683d80319625191918e67d53e1ea79b3b103R3)is correct